test_run.lua

```diff
@@ -437,6 +437,17 @@ local function wait_log(self, node, what, bytes, timeout, opts)
     return wait_cond(self, cond, timeout, delay)
 end
 
+-- Helper for box.snapshot to save changing data in log file instead of
+-- saving in results file, to be able to collect results file checksum
+-- for rerun ability.
+local function box_snapshot()
+    local ok, err = pcall(box.snapshot)
+    if ok == false then
```

Comment:
You can use `not ok`  because `pcall` always returns a bool.

Reply:
Ok, changed.